src/fmu/dataio/_utils.py

```diff
@@ -191,6 +191,21 @@ def read_parameters_txt(pfile):
       LOG10_MULTREGT:MULT_THERYS_VOLON -3.21365
       LOG10_MULTREGT:MULT_VALYSAR_THERYS -3.2582
 
+    Or the structure::
```

Comment:
Explain a bit more the background for this format (and perhaps use the word "format" not "structure"?)